generators/app/templates/_package.json

```diff
@@ -4,12 +4,15 @@
     "main": "index.js",
     "license": "MIT",
     "private": true,
+    "homepage": "<%= homepage %>",
     "devDependencies": {
         "@types/aws-lambda": "^8.10.1",
+        "@types/bunyan": "^1.8.4",
         "@types/faker": "^4.1.1",
         "@types/jest": "^21.1.5",
         "@types/node": "^8.0.47",
         "aws-sdk": "^2.226.1",
+        "bunyan": "^1.8.12",
```

Comment:
Need a 'TODO' for the Bunyan Splunk logger, have a working sample finally...

Reply:
i noticed that splunk have a lambda blueprint for capturing cloudwatch logstreams and sending to their HEC.  could be worth centralizing there.  https://www.splunk.com/blog/2017/02/03/how-to-easily-stream-aws-cloudwatch-logs-to-splunk.html